samples/sapt-sf1/input.dat

```diff
@@ -0,0 +1,28 @@
+#! Tests the Psi4 SF-SAPT code
+
+molecule dimer {
+# Set the spin of A
+0 2
+Li   0.00    0.00    0.00
+--
+# Set the spin of B
+0 2
+H   0.000   0.00   10.00
+symmetry c1
+units bohr
+}
+
+set {
+basis cc-pVDZ
+scf_type pk
+e_convergence 1e-8
+d_convergence 1e-8
+reference rohf
+}
+
+
+
+energy("SF-SAPT")
+
+for k, v in compare_dict.items():
```

Comment:
This is another test that needs `#TEST` somewhere.  I thought you'd found them all before